lib/active_model/serializer/adapter/json_api/api_objects.rb

```diff
@@ -0,0 +1,38 @@
+module ActiveModel
+  class Serializer
+    module Adapter
+      class JsonApi
+        module ApiObjects
+          # Make JSON API top-level jsonapi member opt-in
+          # ref: http://jsonapi.org/format/#document-top-level
+          ActiveModel::Serializer.config.jsonapi_include_toplevel_object = false
+          ActiveModel::Serializer.config.jsonapi_version = '1.0'
+          ActiveModel::Serializer.config.jsonapi_toplevel_meta = {}
```

Comment:
(btw, the reason I didn't originally create this file was because I wasn't sure if I liked the abstraction of `ApiObjects` and wanted it to incubate a bit)